numpy/core/src/multiarray/methods.c

```diff
@@ -310,16 +312,18 @@ array_argmin(PyArrayObject *self,
 {
     int axis = NPY_MAXDIMS;
     PyArrayObject *out = NULL;
+    npy_bool keepdims = NPY_FALSE;
     NPY_PREPARE_ARGPARSER;
 
     if (npy_parse_arguments("argmin", args, len_args, kwnames,
             "|axis", &PyArray_AxisConverter, &axis,
             "|out", &PyArray_OutputConverter, &out,
+            "|keepdims", &PyArray_BoolConverter, &keepdims,
```

Comment:
```suggestion
            "$keepdims", &PyArray_BoolConverter, &keepdims,
```
To make it keyword argument only.